roles/serviceassurance/tasks/component_metrics_smartgateway.yml

```diff
@@ -13,6 +13,7 @@
         service_type: 'metrics'
         size: 1
         prefetch: 15000
+        use_timestamp: 'true'
```

Comment:
The automatic snake_case to camelCase freaks me out every time.

Reply:
Yea I really dislike it...